operator/scripts/prepare-olm-test.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+set -euxo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+DOCKER_REGISTRY="$1"
+
+VERSION="$2"
+PREV_VERSION="$3"
+
+OPERATOR_IMAGE_NAME="keycloak-operator"
+OPERATOR_DOCKER_IMAGE="$DOCKER_REGISTRY/$OPERATOR_IMAGE_NAME"
+
+# Create OLM bundle
+$SCRIPT_DIR/create-olm-bundle.sh $VERSION $PREV_VERSION $OPERATOR_DOCKER_IMAGE
+
+# WARNING: This should be done ONLY for tests
+$SCRIPT_DIR/inject-custom-image.sh $VERSION "$DOCKER_REGISTRY/keycloak:$VERSION"
```

Comment:
No need for this. Env vars can be directly specified in the `Subscription`, see https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md.

Reply:
works, using `subscription.config` and removing the script